_data/investing.yml

```diff
@@ -177,7 +177,9 @@ websites:
       img: vanguard.png
       tfa: Yes
       sms: Yes
-      doc: https://personal.vanguard.com/us/insights/article/Account-security-092015
+      phone: Yes
+      hardware: Yes
```

Comment:
As mentioned in the PR, the information is only available while logged in, here's a screenshot:

![screen shot 2017-01-04 at 8 59 06 pm](https://cloud.githubusercontent.com/assets/587187/21667693/c293af6a-d2c0-11e6-9aa9-ee5b34d0dca3.png)